fmriprep/reports/core.py

```diff
@@ -54,10 +75,52 @@ def generate_reports(subject_list, output_dir, run_uuid, config=None, work_dir=N
             import sys
             import traceback
 
+            # Store the list of subjects for which report generation failed
             errors.append(subject_label)
             traceback.print_exception(
                 *sys.exc_info(),
                 file=str(Path(output_dir) / "logs" / f"report-{run_uuid}-{subject_label}.err"),
             )
 
+        if n_ses >= config.execution.max_ses_agr:
```

Comment:
`max` says to me maximum before something different happens, not the point at which something different happens.

```suggestion
        if n_ses > config.execution.max_ses_agr:
```